script/setup.bat

```diff
@@ -0,0 +1,8 @@
+rem Install all dependencies:
```

Comment:
This should be a separate PR.

Reply:
Yes, that's another commit and got into this PR. Have removed the Windows part.